e2e-playwright/dashboard-cujs/adhoc-filters-cujs.spec.ts

```diff
@@ -0,0 +1,381 @@
+import { test, expect } from '@grafana/plugin-e2e';
+
+import scopesDashboardOne from '../dashboards/scopes-cujs/scopeDashboardOne.json';
+import { setScopes } from '../utils/scope-helpers';
+
+test.use({
+  featureToggles: {
+    scopeFilters: true,
+    groupByVariable: true,
+    reloadDashboardsOnParamsChange: true,
+  },
+});
+
+const USE_LIVE_DATA = Boolean(process.env.USE_LIVE_DATA);
+const LIVE_DASHBOARD_UID = process.env.LIVE_DASHBOARD_UID;
+
+export const DASHBOARD = USE_LIVE_DATA && LIVE_DASHBOARD_UID ? LIVE_DASHBOARD_UID : 'scopes-dashboard-1';
```

Comment:
We can either add UID flags to the tests themselves or when `USE_LIVE_DATA` is enabled we do NOT import the provisioned dashboard and expect that the live instance would have a dashboard with this hardcoded name

Reply:
I think we should remove the UID flag and any live data runs could either already have a db with that uid (which would mean the `api/import` would fail, I think?) or just import it. the removal step tho is going to remove the predefined db with the hardcoded uid tho, so will need some tinkering there